libs/sdk-ui-dashboard/src/presentation/flexibleLayout/DashboardLayoutWidget.tsx

```diff
@@ -194,6 +194,7 @@ export const DashboardLayoutWidget: IDashboardLayoutWidgetRenderer<
                     onError={onError}
                     onFiltersChange={onFiltersChange}
                     widget={widget as ExtendedDashboardWidget}
+                    parentLayoutItem={item}
```

Comment:
This provides parent item to section, section header, and nested layout renderer to know the size they can occupy.